config/default.js

```diff
@@ -82,6 +81,8 @@ module.exports = {
         'Stem Cells and Regenerative Medicine',
       'structural-biology-molecular-biophysics':
         'Structural Biology and Molecular Biophysics',
+      'human-biology': 'Human Biology',
+      medicine: 'Medicine',
```

Comment:
Current order seems to be alphabetical. Should we retain that order?

Reply:
Good spot! Yes we should :-)